src/legacy/core_plugins/vis_type_timelion/public/helpers/timelion_request_handler.ts

```diff
@@ -21,8 +21,8 @@ import { i18n } from '@kbn/i18n';
 import { KIBANA_CONTEXT_NAME } from 'src/plugins/expressions/public';
 import { VisParams } from 'src/legacy/core_plugins/visualizations/public';
 import { TimeRange, Filter, esQuery, Query } from '../../../../../plugins/data/public';
-import { timezoneProvider } from '../legacy_imports';
 import { TimelionVisDependencies } from '../plugin';
+import { getTimezone } from '../../../timelion/public/lib/get_timezone';
```

Comment:
The `timelion` plugin is the legacy app slated to get removed with 8.0 - with should import things from there within the `vis_type_timelion` (which will stay). Looks like it makes sense to move the helper over and import the other way (in `timelion` import from `vis_type_timelion`).

Reply:
thx, will do!